ushahidi/platform-client
# Develop

This pull request makes the following changes:
- Added post review message 
- Adding intercom support which is disabled unless an app id is provided
- Added testing for intercom controller

Test these changes by:
- Add a post to a non-private deployment as a non-logged in user
  - you should see a message which states that the post has been saved and will be reviewed by an admin
- Ensure that intercom does not appear on an OSS deployment

Fixes ushahidi/platform# .

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/135)
